wrapanapi/azure.py

```diff
@@ -544,6 +544,48 @@ def remove_pips_by_search(self, pip_template):
                     Remove-AzureRmPublicIpAddress -Name \"{}\" -ResourceGroupName {} -Force
                 """.format(piname, self.resource_group), ignore_error=True)
 
+    def remove_unused_blobs(self):
```

Comment:
In testing this has run for a full hour with no status logging or anything.

Is it anticipated that it will take this long?  Can the script be broken up so that we get some status as its running - get the list of unmanged vhd's and log, then get list of managed vhds and log, then call delete on each list?  Maybe split the list and thread script calls to Remove-AzureStorageBlob/Remove-AzureRmDisk?